action.yml

```diff
@@ -20,6 +20,10 @@ inputs:
   failure_on_error:
     description: Boolean to make the action report a failure when dbt-cloud runs. Mark this as `false` to run fal after the dbt-cloud job.
     required: true
+  get_artifacts:
+    description: Whether run results, needed by fal, are fetched from dbt cloud. If using this action in other contexts this can be set to `false`, useful for jobs which do not generate artifacts.
```

Comment:
Great description.

Reply:
Ah fantastic, I'm not familiar with fal so I wasn't sure I got this right - forgot to ask for feedback on this in the PR 😄